### PR TITLE
Adds new error status codes.

### PR DESCRIPTION
**Pull request by ngetchell**

### CODE REVIEW

File: PSGitLab/Private/QueryGitLabAPI.ps1 [hunk 1]
```diff
@@ -56,8 +56,12 @@ Function QueryGitLabAPI {
         $totalPages = if ($webContent.Headers.ContainsKey('X-Total-Pages')) {
             (($webContent).Headers['X-Total-Pages']).tostring() -as [int]
         } else { 0 }
-        $bytes = $webContent.Content.ToCharArray() | Foreach-Object{ [byte]$_ }
-        $Results = [Text.Encoding]::UTF8.GetString($bytes) | ConvertFrom-Json
+        if ( $null -eq $webContent.Content ) {
```
Comment (Plork) [inline PSGitLab/Private/QueryGitLabAPI.ps1:59]: Why is this added here? It seems that it checks if the result is "null" and if false .. returns $result = $null

So basically you always get $null when the result is only 1 page.

Comment (ngetchell) [reply PSGitLab/Private/QueryGitLabAPI.ps1:59]: This was added since not all endpoints return anything.
